pom.xml

```
@@ -87,8 +87,14 @@
 			<artifactId>commons-codec</artifactId>
 			<version>1.4</version>
 		</dependency>
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-core</artifactId>
+      <version>1.2-groovy-2.5</version>
```

is it 1.2 or 1.3-groovy-2.5? I have 1.3 for MetaStats.

---

Good question. I did let IntelliJ add it to the pom for me.
```suggestion
      <version>1.3-groovy-2.5</version>
```